docs/nb/Simple_Scripts.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Simple Scripts with SpecDB (v1.2.1)"
+    "# Simple Scripts with SpecDB (v1.3)"
```

Comment:
Note that the 'specdb_plot' script fails because the specdb.py looks for an environment variable called 'SPECDB' (line 478), but the warning thrown says 'Environmental variable IGMSPEC_DB not set.'  Which should the environment variable be set to?  The doc 'installing.rst' refers to 'SPECDB'.